### ISSUE TITLE
Option to specify where local-dynamo should be installed

### ISSUE THREAD

**Issue by martinmicunda** [cvburgess/local-dynamo#1]
Is it possible to add option where `local-dynamo` could be installed? The local-dynamo is installed in user home directory which is okay but I would like to have option to specify where I want to install local-dynamo e.g. in the project root folder so when I decide to delete project I will delete all dynamo jar files.

**Comment by cvburgess** [comment]
@martinmicunda absolutely. [Here's the LOC responsible for defaulting to the home directory.](https://github.com/cvburgess/better-local-dynamo/blob/master/lib/download.js#L13) If you have a suggestion, open a PR and I'd be happy to review and merge it in.
